agenta-cli/agenta/sdk/agenta.py

```diff
@@ -199,3 +244,31 @@ def wrapper(*args, **kwargs):
         print(func(**vars(args)))
 
     return wrapper
+
+
+
+def override_schema_for_multichoice(
```

Comment:
Can you add docstring explaining wjat this does an d why

Reply:
Absolutely!